src/amuse/rfi/channel.py

```diff
@@ -35,8 +35,9 @@
 from amuse.support.options import OptionalAttributes, option, GlobalOptions
 from amuse.support.core import late
 from amuse.support import exceptions
-from amuse.support import get_amuse_root_dir
+from amuse.support import get_amuse_root_dir, get_amuse_directory_root
```

Comment:
these function names are easy to confuse...

Reply:
yea..in amusifier its called --get-amuse-package-dir, maybe thats better?

Reply:
prob consistency is best..